src/components/Experience.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import generateKey from '@tammiec/generatekey';
+
+import './Experience.scss';
+
+const Experience = ({ data, type, orgType, title, ...props }) => {
+
+  const experience = data.map(exp => {
+    
+    const position = type === 'education' ? `${exp.area}, ${exp.studyType}` : exp.position;
+    const summary = type === 'education' ? `GPA: ${exp.gpa}` : exp.summary;
+    const listItems = type === 'education' ? 'courses' : 'highlights';
+
+    return (
+      <div key={generateKey()}>
+        <h3>{exp[orgType]}</h3>
+        {type !== 'education' && <a href={exp.website}>{exp.website}</a>}
+        <p><span>{position}</span> | {exp.startDate} - {exp.endDate}</p>
```

Comment:
Great ! Would be nice if the user can have the date already formatted in a more readable way.

Some libraries for date formatting are: momentjs, date-fns, luxon. I totally recommend dayjs.